dev-util/colm/colm-0.13.0.5.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="COmputer Language Manipulation"
+DESCRIPTION="Colm is a programming language designed for the analysis and \
+transformation of computer languages. Colm is influenced primarily by TXL. It \
+is in the family of program transformation languages."
+HOMEPAGE="https://www.colm.net/open-source/colm/"
+COPYRIGHT="2006-2016 Adrian Thurston"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.colm.net/files/colm/colm-$portVersion.tar.gz"
+CHECKSUM_SHA256="33e624677176958eaad76ebe6c391a68a0b4728fec8cc039efa1316f525f408c"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandSuffix=$secondaryArchSuffix
+	commandBinDir=$binDir
+else
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	colm$secondaryArchSuffix = $portVersion
+	cmd:colm$commandSuffix = $portVersion compat >= 1.13
+	lib:libcolm_$portVersion$secondaryArchSuffix = $portVersion compat >= 1.13
```

Comment:
I think `compat` version for the library quite useless since the version is already in it's name. Also, the lib doesn't seem to provide any kind of ABI stability

```diff
+binVersionCompat="$portVersion compat >= ${portVersion%.*.*}"
 
 PROVIDES="
-	cmd:colm$commandSuffix = $portVersion compat >= 1.13
-	lib:libcolm_$portVersion$secondaryArchSuffix = $portVersion compat >= 1.13
+	cmd:colm$commandSuffix = $binVersionCompat
+	lib:libcolm_$portVersion$secondaryArchSuffix = $portVersion